### PR TITLE
Coveralls

### PR DESCRIPTION
**Pull request by mayabrandi**
I don't know how to test what Im trying to set up. But maybe you can tell if it is going to work just by looking at it? @ingkebil

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,16 @@ python:
...
-  - pip install pytest
-  - pip install --editable .
+    - pip install pytest
+    - pip install pytest-cov
+    - pip install coveralls
+    - pip install --editable .
 
 script:
-  - py.test -v tests/*
+    - py.test -v tests/*
```
Comment (northwestwitch) [inline .travis.yml]: You could add --cov=vogue , to avoid statistics on stuff which is not inside the main app.

**Comment by ingkebil** [comment]
In your build (click "Show all check", pick one of the "details" links), at the bottom it will show following link: https://coveralls.io/jobs/47367384

That seems to be empty, hence not working.

**Comment by ingkebil** [comment]
Also, please have a look at how `cg` has set up travis and coveralls: https://github.com/Clinical-Genomics/cg/blob/master/.travis.yml

**Comment by hassanfa** [comment]
Fixing this in another branch.

**Pull request closed without merging**